src/Front/PayplugOney/Requests/OneyBase.php

```diff
@@ -123,17 +123,18 @@ public function showOneyAnimationProduct()
 	{
 		if ( (is_product()) && PayplugWoocommerceHelper::is_oney_available()) {
 			global $product;
-
 			$total_price = (is_numeric( floatval(WC()->cart->total))) ? floatval(WC()->cart->total) : (float)($product->get_price());
+			$price = $product->get_price();
```

Comment:
you've to confirm with @zadil99 which price is used to enable/disable the  popup.

you're adding the value of the product and we were using only the cart value. On magento we are using only the product price... we need more details for this